### ISSUE TITLE
Lenovo ideapad S-130; removing ideapad mod switches off keyboard

### ISSUE THREAD

**Issue by recumbentbirder** [tomaspinho/rtl8821ce#23]
_Your_ module does precicely what it's intended to (I guess: all the WiFi networks are shown then): after removing the ideapad_laptop module as you describe though, I cannot use my keyboard anymore, therefore not even enter the WiFi password.

How can I help you help me? -- or maybe this problem is even out of scope of your driver?

**Comment by recumbentbirder** [comment]
PS. confirmed with external keyboard: the ideapad_laptop module is needed for the internal _keyboard_. Touchpad (and external keyboards) work, though.

**Comment by recumbentbirder** [comment]
Now I have keyboard and WiFi running at the same time. Don't know exactly why. Might have to do with me using `rfkill unblock wlan` as suggested on [the Archlinux Wiki](https://wiki.archlinux.org/index.php/Lenovo_IdeaPad_Yoga_2_Pro).

But seems to be far away enough for it being not your concern (unless you have some hints / tips), so that I close the issue again.

Sorry, and lots of thanks for this module!
